src/footer.stories.mdx

```diff
@@ -0,0 +1,144 @@
+import { Stack } from './Stack';
+import { Grid, GridItem } from './Grid';
+import { FooterPartner } from './FooterPartner';
+import { FooterList, FooterListItem } from './FooterList';
+import { StoreMenu } from './StoreMenu';
+import styled from 'styled-components';
+import { fontFamily, fontWeight, space, color, borderRadius } from '@core-ds/primitives';
+import stores from './stores.json';
+import languages from './languages.json';
+import links from './links.json';
+import partners from './partners.json';
+import { Globe, FacebookLogo, InstagramLogo, TwitterLogo, YoutubeLogo, RepairOrgLogo } from './svg';
+
+<Meta title="Footer" />
+
+export const DemoPage = styled.div`
+   display: flex;
+   flex-direction: column;
+   font-family: ${fontFamily.inter};
+   margin-top: 100px;
+`;
+
+# iFixit footer
+
+A set of components to compose iFixit.com and Shopify store footer.
+
+<Canvas>
+   <Story name="Default footer">
+      {(args) => (
+         <DemoPage>
+            <Footer>
+               <Grid
+                  templateColumns={{ base: '1fr', sm: 'repeat(3, 1fr)', lg: 'repeat(4, 1fr)' }}
+                  gap={space[4]}
+                  paddingX={{
+                     base: space[5],
+                     sm: space[7],
+                  }}
+                  paddingY={space[7]}
+               >
+                  {links.map((list, index) => (
+                     <FooterList key={index}>
+                        {list.map((link, linkIndex) => (
+                           <FooterListItem key={linkIndex}>
+                              <Link href={link.url} fontWeight={fontWeight.bold}>
+                                 {link.label}
+                              </Link>
+                           </FooterListItem>
+                        ))}
+                     </FooterList>
+                  ))}
+                  <FooterList>
+                     <FooterListItem>
+                        <Link href="#" prefix={<FacebookLogo />}>
+                           Facebook
+                        </Link>
+                     </FooterListItem>
+                     <FooterListItem>
+                        <Link href="#" prefix={<TwitterLogo />}>
+                           Twitter
+                        </Link>
+                     </FooterListItem>
+                     <FooterListItem>
+                        <Link href="#" prefix={<InstagramLogo />}>
+                           Instagram
+                        </Link>
+                     </FooterListItem>
+                     <FooterListItem>
+                        <Link href="#" prefix={<YoutubeLogo />}>
+                           Youtube
+                        </Link>
+                     </FooterListItem>
+                     <FooterListItem>
+                        <Link href="#" prefix={<RepairOrgLogo />}>
+                           Repair.org
+                        </Link>
+                     </FooterListItem>
+                  </FooterList>
```

Comment:
Do you mean something like this?

```tsx

// SocialLogo.tsx
function SocialLogo({ name }: SocialLogoProps) {
  switch (name.toLowerCase()) {
    case "facebook":
      return <FacebookLogo />
    case "Twitter":
      return <TwitterLogo />
    // ...
  }
}

// Footer.tsx
// ...
<FooterList>
  {socials.map(social => (
    <FooterListItem key={social.name}>
      <Link href={social.link} prefix={<SocialLogo name={social.name} />}>
        {social.name}
      </Link>
    </FooterListItem>
  ))}
</FooterList>
// ...
```

I'm not too convinced about this, as you lose a little bit of flexibility, because every time you add a new social you need to update the `SocialLogo` component and publish a new version of the package. 

What do you think? Do you want me to refactor like this anyway?

Reply:
No, I think it's fine for now.